fury/utils.py

```diff
@@ -160,6 +160,43 @@ def numpy_to_vtk_cells(data, is_coords=True):
     return cell_array
 
 
+def numpy_to_vtk_image_data(array, spacing=(1.0, 1.0, 1.0),
+                            origin=(0.0, 0.0, 0.0), depth=1):
```

Comment:
I was talking about the `deep` option and not `depth`. This is a boolean value. See below.

Reply:
Sorry for that, I just got confused between those!